coldfront/core/allocation/views.py

```diff
@@ -232,22 +232,25 @@ def post(self, request, *args, **kwargs):
             if action != 'auto-approve':
                 messages.success(request, 'Allocation Activated!')
 
-        elif old_status != allocation_obj.status.name in ['Denied', 'New']:
+        elif old_status != allocation_obj.status in ['Denied', 'New', 'Revoked']:
```

Comment:
This needs to be `elif old_status != allocation_obj.status.name in ['Denied', 'New', 'Revoked']:`